airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/DefaultGeographyMapper.java

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Here's the OSS implementation of the GeographyMapper. In Cloud, this class will be overridden with a `@Replaces` annotation

Reply:
For my own understanding too, since I haven't had a chance to work much with micronaut (need to change that!).

How do the @Replace and @Primary annotations interact?

Reply:
@davinchia They are two different options for the same result.  Any bean marked `@Primary` will always win.  `@Replace` is used to take the place of an existing bean of the same type.  I haven't played with it, but I suspect that `@Replace` wouldn't work if no other bean exists of that type, but `@Primary` would.  Generally, one is a bit more loose than the other, so it really depends on if we have more than 2 versions, as we won't want to mark two of the three of the same type with `@Primary`.

Reply:
Sounds like we should default to @Primary and use @Replace on a needs-to basis

Reply:
@davinchia That is my assessment as well.